### DISCUSSION TITLE
Allow multiple results per row, for people layout

### DISCUSSION THREAD

**Discussion by gabbsmo** [microsoft-search/pnp-modern-search#3356] [General]
I really like how the card layout can be used to display multiple results per row. This makes efficient use of screen space. I wish that I could do the same with the people layout.

**Comment by kasperbolarsen** [comment]
Have a look at https://github.com/microsoft-search/pnp-modern-search/discussions/3334 😁

**Reply by gabbsmo** [reply]
Thanks! I altered the CSS slightly to make it more in line with the Cards layout.

```
mgt-person {
  --initials-background-color: {{@root.theme.palette.themePrimary}};
  max-width: 360px;
}
.template--defaultList {
  flex-wrap: wrap;
  display: flex;
  justify-content: space-between;
  gap: 16px 8px;    
}  
```

With that said I opened this as a feature request because I curious as to why the cards layout has this functionality but not people. If it is a conscious design decision by the project I think the reason for this should be documented.

**Reply by kasperbolarsen** [reply]
I am not sure about the reason, but in a lot of cases it makes more sense with a single column layout, like in https://www.m365thinking.com/post/using-the-pnp-modern-search-results-web-part-as-a-department-web-part and similar. So I'll vote to keep it that way :-)

**Comment by nthoman** [comment]
I would like to see a multi column option too like the old school SharePoint Directory template out there. Been using that one for years but it was just missing Teams presence. Now that you can show presence with the new version.... would love to be able do show that in Card view also.

**Reply by kasperbolarsen** [reply]
We would love to see a PR to that effect 👍

Personally, I might look into an updated people search experience using Microsoft search once custom user profile profiles (not in the SharePoint User Profile application) will be available.
